src/navigations/HomeBottomTabs.js

```diff
@@ -20,10 +20,12 @@ import {InitialStartupAtom, otherProfileAtom} from '../service/initialStartup';
 import {colors} from '../utils/colors';
 import {fcmTokenService} from '../service/users';
 import {getAnonymousUserId, getUserId} from '../utils/users';
+import useCoreChatSystemHook from '../hooks/core/useCoreChatSystemHook';
 
 const Tab = createBottomTabNavigator();
 
 function HomeBottomTabs({navigation}) {
+  useCoreChatSystemHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useCoreChatSystemHook` is being called without any parameters or without being assigned to a variable. If this hook returns a value, it should be assigned to a variable. If it requires parameters, they should be passed in. Please ensure that the hook is used correctly.
```diff
-  useCoreChatSystemHook();
+  const result = useCoreChatSystemHook(params);
```
Please replace `params` with the actual parameters if any are needed.


<!-- This is an auto-generated comment by OSS CodeRabbit -->